features/footer.feature

```diff
@@ -0,0 +1,8 @@
+Feature: Restaurant Information
+  As a visitor,
+  In order to contact the restaurant
+  I would like to be able to see the contact Information
+
+  Scenario: Viewing restaurant contact information on the application's landing page
+      When I'm on the landing page
+      Then I should see "Phone"
```

Comment:
please add the other things the user should see

Reply:
Done.